src/Symfony/Component/Console/Input/ArgvInput.php

```diff
@@ -348,11 +348,30 @@ public function getParameterOption(string|array $values, string|bool|int|float|a
     /**
      * Returns un-parsed and not validated tokens.
      *
+     * @param bool $strip Whether to return the raw parameters (false) or the values after the command name (true)
+     *
      * @return list<string>
      */
-    public function getRawTokens(): array
+    public function getRawTokens(bool $strip = false): array
     {
-        return $this->tokens;
+        if (!$strip) {
+            return $this->tokens;
+        }
+
+        $parameters = [];
+        $keep = false;
+        foreach ($this->tokens as $value) {
+            if ($value === $this->getFirstArgument()) {
```

Comment:
Shouldn't we do that only if `$keep` is false ? Maybe another argument later will have the same value than the command name.

Reply:
Good catch. I also added a test for this